### DISCUSSION TITLE
My org wants to contribute financially.

### DISCUSSION THREAD

**Discussion by srameshr** [plausible/analytics#652] [Anything Else]
We have ditched Google Analytics and running a self-hosted version of Plausible on our domain. How do we go about having a billing cycle for our self-hosted version so that the open-source project keeps up with features and support?

**Comment by metmarkosaric** [comment]
That's nice to hear, thanks! You can contribute using the sponsor button here on GitHub. Direct link https://github.com/sponsors/plausible

**Comment by ukutaht** [comment]
Thanks @srameshr !
